External/FEXCore/Source/Interface/IR/Passes/ConstProp.cpp

```diff
@@ -198,6 +199,16 @@ class ConstProp final : public FEXCore::IR::Pass {
 
   std::unordered_map<uint64_t, OrderedNode*> ConstPool;
   std::map<OrderedNode*, uint64_t> AddressgenConsts;
+
+  // Pool inline constant generation. These are typically very small and pool efficiently.
+  tsl::robin_map<uint64_t, OrderedNode*> InlineConstantGen;
+  OrderedNode *CreateInlineConstant(IREmitter *IREmit, uint64_t Constant) {
+    if (InlineConstantGen[Constant]) {
+      return InlineConstantGen[Constant];
+    }
+    auto InlineConstant = InlineConstantGen[Constant] = IREmit->_InlineConstant(Constant);
```

Comment:
I think robin_map has the same API as the std for this, so maybe this could be:

```cpp
auto res = InlineConstantGen.insert_or_assign(Constant, IREmit->_InlineConstant(Constant));
return res.first->second;
```

or something like that